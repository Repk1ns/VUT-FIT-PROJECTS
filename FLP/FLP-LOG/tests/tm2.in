S   A R
A a B a
A b B b
A c B c
B b B R
B c B R
B x B R
B a C x
C a C R
C b C R
C c C R
C x C R
C   D L
D a D L
D c D L
D x D L
D b E x
E a E L
E b E L
E c E L
E x E L
E   G R
G a G R
G b G R
G x G R
G c H x
H a H L
H b H L
H c H L
H x H L
H   I R
I x I R
I a J L
I b J L
I c J L
J a J L
J b J L
J c J L
J x J L
J   B R
I   F  
A   F  
 aaabbbccc