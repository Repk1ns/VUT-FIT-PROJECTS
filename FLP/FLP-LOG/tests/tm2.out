S aaabbbccc
 Aaaabbbccc
 Baaabbbccc
 Cxaabbbccc
 xCaabbbccc
 xaCabbbccc
 xaaCbbbccc
 xaabCbbccc
 xaabbCbccc
 xaabbbCccc
 xaabbbcCcc
 xaabbbccCc
 xaabbbcccC
 xaabbbccDc
 xaabbbcDcc
 xaabbbDccc
 xaabbDbccc
 xaabbExccc
 xaabEbxccc
 xaaEbbxccc
 xaEabbxccc
 xEaabbxccc
 Exaabbxccc
E xaabbxccc
 Gxaabbxccc
 xGaabbxccc
 xaGabbxccc
 xaaGbbxccc
 xaabGbxccc
 xaabbGxccc
 xaabbxGccc
 xaabbxHxcc
 xaabbHxxcc
 xaabHbxxcc
 xaaHbbxxcc
 xaHabbxxcc
 xHaabbxxcc
 Hxaabbxxcc
H xaabbxxcc
 Ixaabbxxcc
 xIaabbxxcc
 Jxaabbxxcc
J xaabbxxcc
 Bxaabbxxcc
 xBaabbxxcc
 xCxabbxxcc
 xxCabbxxcc
 xxaCbbxxcc
 xxabCbxxcc
 xxabbCxxcc
 xxabbxCxcc
 xxabbxxCcc
 xxabbxxcCc
 xxabbxxccC
 xxabbxxcDc
 xxabbxxDcc
 xxabbxDxcc
 xxabbDxxcc
 xxabDbxxcc
 xxabExxxcc
 xxaEbxxxcc
 xxEabxxxcc
 xExabxxxcc
 Exxabxxxcc
E xxabxxxcc
 Gxxabxxxcc
 xGxabxxxcc
 xxGabxxxcc
 xxaGbxxxcc
 xxabGxxxcc
 xxabxGxxcc
 xxabxxGxcc
 xxabxxxGcc
 xxabxxxHxc
 xxabxxHxxc
 xxabxHxxxc
 xxabHxxxxc
 xxaHbxxxxc
 xxHabxxxxc
 xHxabxxxxc
 Hxxabxxxxc
H xxabxxxxc
 Ixxabxxxxc
 xIxabxxxxc
 xxIabxxxxc
 xJxabxxxxc
 Jxxabxxxxc
J xxabxxxxc
 Bxxabxxxxc
 xBxabxxxxc
 xxBabxxxxc
 xxCxbxxxxc
 xxxCbxxxxc
 xxxbCxxxxc
 xxxbxCxxxc
 xxxbxxCxxc
 xxxbxxxCxc
 xxxbxxxxCc
 xxxbxxxxcC
 xxxbxxxxDc
 xxxbxxxDxc
 xxxbxxDxxc
 xxxbxDxxxc
 xxxbDxxxxc
 xxxDbxxxxc
 xxxExxxxxc
 xxExxxxxxc
 xExxxxxxxc
 Exxxxxxxxc
E xxxxxxxxc
 Gxxxxxxxxc
 xGxxxxxxxc
 xxGxxxxxxc
 xxxGxxxxxc
 xxxxGxxxxc
 xxxxxGxxxc
 xxxxxxGxxc
 xxxxxxxGxc
 xxxxxxxxGc
 xxxxxxxxHx
 xxxxxxxHxx
 xxxxxxHxxx
 xxxxxHxxxx
 xxxxHxxxxx
 xxxHxxxxxx
 xxHxxxxxxx
 xHxxxxxxxx
 Hxxxxxxxxx
H xxxxxxxxx
 Ixxxxxxxxx
 xIxxxxxxxx
 xxIxxxxxxx
 xxxIxxxxxx
 xxxxIxxxxx
 xxxxxIxxxx
 xxxxxxIxxx
 xxxxxxxIxx
 xxxxxxxxIx
 xxxxxxxxxI
 xxxxxxxxxF
